standalone-metastore/metastore-server/src/main/sql/hive/hive-schema-4.1.0.hive.sql

```diff
@@ -1215,7 +1215,7 @@ SELECT
   CC_DATABASE,
   CC_TABLE,
   CC_PARTITION,
-  CASE WHEN CC_TYPE = 'i' THEN 'minor' WHEN CC_TYPE = 'a' THEN 'major' ELSE 'UNKNOWN' END,
+  CASE WHEN CC_TYPE = 'i' THEN 'minor' WHEN CC_TYPE = 'a' THEN 'major' WHEN CC_TYPE = '*' THEN 'smart-optimize' ELSE 'UNKNOWN' END,
```

Comment:
what about upgrade script?

Reply:
Fixed, added creation/replacement of `COMPACTIONS` view to the upgrade script.